app/main.py

```diff
@@ -2,0 +1,25 @@
+from __future__ import annotations
+
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    vaccine_error_msg = []
+    mask_error_msg = []
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError as e:
+            print("You have some issue", e)
+            vaccine_error_msg.append(e)
+        except NotWearingMaskError as e:
+            print(e)
```

Comment:
Consider making the print statement more informative, such as `print(f"Mask error for {friend['name']}: {e}")`.